src/collections/_documentation/platforms/python/bottle.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Bottle
+sidebar_order: 5
+---
+
+{% version_added 0.7.9 %}
+
+
+<!-- WIZARD -->
+*Import name: `sentry_sdk.integrations.bottle.BottleIntegration`*
+
+The Bottle integration adds support for the [Bottle Web Framework](https://bottlepy.org/).
+
+1. Install `sentry-sdk` from PyPI with the `bottle` extra:
+
+    ```bash
+    $ pip install --upgrade 'sentry-sdk[bottle]=={% sdk_version sentry.python %}'
+    ```
+
+2.  To configure the SDK, initialize it with the integration before your app has been initialized:
+
+    ```python
+    import sentry_sdk
+
+    from bottle import Bottle, run
+    from sentry_sdk.integrations.bottle import BottleIntegration
+
+    sentry_sdk.init(
+        dsn="___PUBLIC_DSN___",
+        integrations=[BottleIntegration()]
+    )
+
+    app = Bottle()
+    ```
+
+<!-- ENDWIZARD -->
+
+## Behavior
+
+* The Sentry Python SDK will install the Bottle integration for all of your apps. The integration hooks into base Bottle class.
+
+* All exceptions leading to an Internal Server Error are reported.
+
+* {% include platforms/python/request-data.md %}
+
+## Options
+
+You can pass the following keyword arguments to `BottleIntegration()`:
+
+* `transaction_style`:
+
+  ```python
+  @app.route("/myurl/<foo>")
+  def myendpoint():
+      return "ok"
+  ```
+
+  In the above code, you would set the transaction to:
+
+  * `/myurl/<foo>` if you set `transaction_style="url"`. This matches the behavior of the old Raven SDK.
+  * `myendpoint` if you set `transaction_style="endpoint"`
+
+  The default is `"endpoint"`.
+
+## User Feedback
+
+You can use the user feedback feature with this integration.  For more information see [User Feedback]({% link _documentation/enriching-error-data/user-feedback.md %}?platform=bottle).
```

Comment:
This link will not work. If you want to make an example for user feedback, there's the file `_site/documentation/enriching-error-data/user-feedback-example/flask.md` that you'd need to copy. But we didn't do this for all web frameworks either.

Reply:
Ok, I removed it for now.